httpserver/src/main/kotlin/tbdex/sdk/httpserver/TbdexHttpServer.kt

```diff
@@ -134,36 +134,74 @@ class TbdexHttpServer(val config: TbdexHttpServerConfig) {
           getExchange(
             call,
             exchangesApi,
-            getCallbacks.getOrDefault("exchange", null),
+            callbacks.getOrDefault("exchange", null) as GetExchangeCallback?,
             pfiDid
           )
         }
       }
 
       get("/offerings") {
-        getOfferings(call, offeringsApi,  getCallbacks.getOrDefault("offerings", null))
+        getOfferings(
+          call,
+          offeringsApi,
+          callbacks.getOrDefault("offerings", null) as GetOfferingsCallback?
+        )
       }
     }
   }
 
   /**
-   * Adds a submit callback for a specific message kind (RFQ, order, close, etc.).
+   * Adds a GetOfferingsCallback for handling requests for offerings.
+   *
+   * @param callback GetOfferingsCallback function to be registered
+   */
+  fun getOfferings(callback: GetOfferingsCallback) {
```

Comment:
I think it may be clearer (but maybe not Kotlin-y) to change the names of these functions to `onGetOfferings()` etc.

Reply:
hmm, so you'd prefer

```kt
  tbdexServer.onGetOfferings { call ->
    println("Getting offerings...")
    call.respond(HttpStatusCode.OK, listOf<Offering>())
  }
```

over

```kt
  tbdexServer.getOfferings { call ->
    println("Getting offerings...")
    call.respond(HttpStatusCode.OK, listOf<Offering>())
  }
```

?

Reply:
@jiyoontbd Personally I also like `tbdexServer.onGetOfferings`, but it doesn't matter too much either way. I trust your judgement

Reply:
actually, i think i will do `on...` because else i keep confusing it with the handler methods which have the same names. fixed in https://github.com/TBD54566975/tbdex-kt/pull/218/commits/ab8de7e473890207bbc95a3a0b9a42c279a4ca59